api/src/routes/user.ts

```diff
@@ -49,5 +49,69 @@ export const userRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.post(
+    '/account/reset-progress',
+    {
+      schema: {
+        response: {
+          200: Type.Object({}),
+          500: Type.Object({
+            message: Type.Literal(
+              'Oops! Something went wrong. Please try again in a moment or contact support@freecodecamp.org if the error persists.'
+            ),
+            type: Type.Literal('danger')
+          })
```

Comment:
Could you add this to `schemas.ts`? Also, if you're feeling keen this gets used a lot so there's room for DRYing.

Reply:
I think that it's better to DRY them out in the end just before the MVP release as by then all(or most) routes schemas will be in one place and we can DRY it out in one go

Reply:
Sure, no rush.